### PR TITLE
fix submissions table

### PR DESCRIPTION
**Pull request by Searay-330**
reworked table to address #44 and #42 as it turns out that entire table had features we didnt know about it just didnt work at all, side note this also contains a jquery version update so we can leave 2014 jquery behind

### CODE REVIEW

File: app/assets/javascripts/manage_submissions.js [hunk 1]
```diff
@@ -45,17 +44,16 @@ jQuery(function($) {
...
       [3, "desc"]
+    ],
+    "aoColumnDefs": [
+      { "bSortable": false, "aTargets": [ 0 ] },
+      {"bVisible": false, "aTargets": [7]}
     ]
   });
 
-  $("#only-latest").on("change", function() {
+  $("#only-latest").on("change", function() {;
```
Comment (daviddob) [inline app/assets/javascripts/manage_submissions.js]: What is this semicolon for?

File: app/assets/javascripts/table_floating_header.js [hunk 2]
```diff
@@ -1,147 +1,147 @@
-var floating_header = function() {
+// var floating_header = function() {
```
Comment (daviddob) [inline app/assets/javascripts/table_floating_header.js]: Why comment all this out instead of just pulling it?

Comment (Searay-330) [reply app/assets/javascripts/table_floating_header.js]: i was thinking about links needed to resolve but ill pull it

File: app/assets/javascripts/manage_submissions.js (hunk 1 above)
Comment (Searay-330) [reply app/assets/javascripts/manage_submissions.js]: ends that nonexistent line, its ok in JS random semicolons are a feature

File: app/assets/javascripts/table_floating_header.js [hunk 3]
```diff
@@ -1,147 +0,0 @@
-var floating_header = function() {
```
Comment (daviddob) [inline app/assets/javascripts/table_floating_header.js]: Are we certain this function for tables isn't used anywhere else in the codebase?

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -35,7 +35,7 @@
       js.src = "//connect.facebook.net/en_US/sdk.js#xfbml=1&version=v2.3&appId=259908887444831";
       fjs.parentNode.insertBefore(js, fjs);
     }(document, 'script', 'facebook-jssdk'));</script>
-    <%= external_javascript_include_tag "jquery", "2.1.4" %>
```
Comment (daviddob) [inline app/views/layouts/application.html.erb]: Was backwards compatibility broken between 2.x and 3.x? Could this cause issues elsewhere in the frontend?

File: app/assets/javascripts/table_floating_header.js (hunk 3 above)
Comment (Searay-330) [reply app/assets/javascripts/table_floating_header.js]: ill give this a 2nd pass i couldnt locate any on the first trip

File: app/views/layouts/application.html.erb (hunk 4 above)
Comment (Searay-330) [reply app/views/layouts/application.html.erb]: as people have pointed out there isnt alot of JS on the site to begin with and everything i wrote is compatible. the tables being used the way they currently are were not supported by jquery 2. ill give this another pass on the way by. but the very little js on the site worked fine on pass 1.

Comment (Searay-330) [reply app/views/layouts/application.html.erb]: everything seems to be in tip top shape and working as i would expect.

**Pull request merged**